### ISSUE TITLE
06_functions exercise: Same court name returned for all cases

### ISSUE THREAD

**Issue by danmichaelo** [scriptotek/programming-for-lawyers#10]
In "Exercise: Functions to get Information", we loop over three cases, but all three are from the same court. This makes it a bit harder to distinguish a correct solution from a wrong one. For instance, a learner got the "correct solution" from this attempt:

```
def get_court_name(case):
    return court["name"]
```

because court was defined earlier in the Notebook.

We should try to replace the API call with one that returns cases from different courts.

**Comment by danmichaelo** [comment]
Perhaps just remove `jurisdiction=ill`? Then it returns

```
North Carolina Court of Appeals
United States Court of Federal Claims
United States District Court for the District of Columbia
```

**Comment by ewinge** [comment]
fixed in ee173355dd11e561e2267421afbefeba33f89e3c
